relay/cmd/start.go

```diff
@@ -0,0 +1,69 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ipfs/go-log"
+	"github.com/keep-network/tbtc/relay/config"
+	"github.com/keep-network/tbtc/relay/pkg/btc"
+	"github.com/keep-network/tbtc/relay/pkg/chain"
+	"github.com/keep-network/tbtc/relay/pkg/chain/ethereum"
+	"github.com/keep-network/tbtc/relay/pkg/forwarder"
+	"github.com/urfave/cli"
+)
+
+var logger = log.Logger("relay-cmd")
+
+const startDescription = `
+Starts the relay maintainer in the foreground.
+
+It requires the password of the operator host chain key file to be provided
+as ` + config.PasswordEnvVariable + ` environment variable.
+`
+
+// StartCommand contains the definition of the start command-line sub-command.
+var StartCommand = cli.Command{
+	Name:        "start",
+	Usage:       `Starts the relay maintainer in the foreground`,
+	Description: startDescription,
+	Action:      Start,
+}
+
+// Start starts the relay maintainer.
+func Start(c *cli.Context) error {
+	ctx := context.Background()
+
+	config, err := config.ReadConfig(c.GlobalString("config"))
+	if err != nil {
+		return fmt.Errorf("could not read config file: [%v]", err)
+	}
+
+	btcChain, err := btc.Connect()
+	if err != nil {
+		return fmt.Errorf("could not connect BTC chain: [%v]", err)
+	}
+
+	hostChain, err := connectHostChain(ctx, config)
+	if err != nil {
+		return fmt.Errorf("could not connect host chain: [%v]", err)
+	}
+
+	err = forwarder.Initialize(btcChain, hostChain)
```

Comment:
`forwareder` or `node`? This is relay node.

Reply:
For now, it's just a forwarder and anything else. During development, we will probably introduce a `node` layer to handle relay modes and stuff. I just wanted to avoid running too far in the future at the moment.

Reply:
I'd suggest doing `note.Initialize` to keep it aligned with other projects. It is much easier to navigate the structure this way. And inside `node.Initialize` we can have a call starting a forwarder loop that is implemented in `forwarder` package.

Reply:
Ok, done in https://github.com/keep-network/tbtc/commit/a5c9973b416081f84f2cf58e214aa9da6e9c37a4